libs/GraphicECS/SFML/Systems/InputManagement.cpp

```diff
@@ -90,7 +92,7 @@ void InputManagement::run(World &world)
         }
     }
     for (auto &entityPtr : Inputs) {
-        std::queue<std::pair<ecs::ActionQueueComponent::inputAction_e, float>> &actions =
+        std::queue<std::pair<graphic::sfml::ActionQueueComponent::inputAction_e, float>> &actions =
```

Comment:
don't use "graphic::sfml" if you write "using namespace graphic::sfml" at the beginning of the file